### ISSUE TITLE
[TSDoc> Search]: Keyboard focus is not visible on the 'Clear the query' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#244]
**Test Environment:** 
Version: Version 24H2 (OS Build 26231.5000) 
Browser Version: 127.0.2638.2 (Official build) dev (64-bit) 
URL: https://tsdoc.org/ 
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through Ctrl+ windows+ Enter key. 
3. Navigate to the 'Search' button and press enter key to activate it.
4. Type some characters, press tab key, and observe the issue.

**Actual Result:**
While navigating to 'Clear the query' button using keyboard, Keyboard focus is not visible on the button. When the focus lands on it then only tooltip is getting appears.

**Expected Result:**
Keyboard focus should be properly visible on the 'Clear the query' button.

**User Impact:**
It will impact the keyboard user as they will not able to know about the focused control if the keyboard focus is not visible.

**WCAG Reference:**
Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

![Keyboard focus is not visible on the clear the query button](https://github.com/microsoft/rushstack-websites/assets/95913748/67df73b0-b922-4803-b0ce-6890d022072f)

https://github.com/microsoft/rushstack-websites/assets/95913748/61c23974-c15d-4687-b69d-7937c4365210
